plugins/outputs/sql/sql_test.go

```diff
@@ -211,31 +211,32 @@ func TestMysqlIntegration(t *testing.T) {
 		testMetrics,
 	))
 
-	//dump the database
-	var rc int
-	rc, _, err = container.Exec([]string{
-		"bash",
-		"-c",
-		"mariadb-dump --user=" + username +
-			" --password=" + password +
-			" --compact --skip-opt " +
-			dbname +
-			" > /out/dump",
-	})
-	require.NoError(t, err)
-	require.Equal(t, 0, rc)
-	dumpfile := filepath.Join(outDir, "dump")
-	require.Eventually(t, func() bool {
-		_, err := os.Stat(dumpfile)
-		return !os.IsNotExist(err)
-	}, 5*time.Second, 10*time.Millisecond)
+	cases := []struct {
+		expected string
+	}{
+		{"INSERT INTO `metric_one` VALUES ('2021-05-17 22:04:45','tag1','tag2',1234,2345,1,0,1000000000,3.1415);"},
+		{"INSERT INTO `metric_two` VALUES ('2021-05-17 22:04:45','tag3','string1');"},
+		{"INSERT INTO `metric three` VALUES ('2021-05-17 22:04:45','tag4','string2');"},
+	}
+	for _, tc := range cases {
```

Comment:
This table test doesn't look related to switching from file to reading stdout from the exec. Did it get migrated from somewhere else like a data file? Do we need to remove it from somewhere else?